### ISSUE TITLE
Easier access to the model representation of one Group

### ISSUE THREAD

**Issue by LilithWittmann** [serioeseGmbH/serious-django-permissions#2]
There are some cases when you need the model representation of a Group. Currently we have to write something like:
 
```python
self.user.groups.add(AuthorizedGroup.get_or_create()[0].pk)
# or
self.user.groups.add(Group.objects.filter(name=AuthorizedGroup.group_name).pk)
```
to add a user to a group.

It would be much cooler if there would be an easier option to access the model  representation of a Group Object.

```python
# maybe something like
self.user.groups.add(AuthorizedGroup)

# or
self.user.groups.add(AuthorizedGroup.model.pk)

# or

self.user.groups.add(AuthorizedGroup.group_id)

```

What do you think @cobalamin?

**Comment by seblat** [comment]
@LilithWittmann and me have just tried some things. A possible solution could be to extend the `Group` class with this method:
```
@classmethod
    def id(cls):
        return DjangoGroup.objects.get(name=cls.group_name).pk
```

**Comment by cobalamin** [comment]
That looks decent to me, which of the calls suggested by @LilithWittmann would that enable? The first one, `self.user.groups.add(AuthorizedGroup)`?

**Comment by seblat** [comment]
> That looks decent to me, which of the calls suggested by @LilithWittmann would that enable? The first one, `self.user.groups.add(AuthorizedGroup)`?

This would enable `user.groups.add(AuthorizedGroup.id())`.

I think `user.groups.add(AuthorizedGroup)` would be even nicer. Yet so far I haven't found a way to alter the Python return when calling `AuthorizedGroup`. So far this returns the AuthorizedGroup Class Object. Also I'm not sure if we want to do this from a Python POV.

**Comment by cobalamin** [comment]
Right, that'd be what one would want and expect. I'll have a look soon.

**Comment by cobalamin** [comment]
@LilithWittmann @seblat This works now, released in 0.9 :) All I had to do was to make the classes convertible to integers (their primary keys) by implementing `__int__(cls)` on the metaclass.
